content/en/docs/concepts/logic.md

```diff
@@ -492,30 +515,36 @@ it is guaranteed that this will be the final result.
 If a value is not concrete (like `string`), it is clear the search
 for a concrete value is not over.
 In other words, an instance may never violate the constraints of its parent.
-This property makes it very hard to inadvertendly make false conclusions in CUE.
+This property makes it very hard to inadvertently make false conclusions in CUE.
 Default values do not change this property; they syntactically appear as
 non-concrete values.
 CUE also bails out and requires explicit values if two conflicting defaults
 are specified for the same field, again limiting the search space.
+<!-- (jba) There's no actual searching going on in the implementation (IIUC), so using the term "search
+space" is confusing. -->
```

Comment:
The search applies to users trying to track dow the origin of a value, not CUE's evaluation.